script/Arcueid/Day 7/07_00_ARC07_3_2.txt

```diff
@@ -21,27 +21,27 @@ Besides, one could say that I've been dragged around by her since the evening be
 [sha:f8d483e99d31dfc501858f1098c3a6d0c16fdd54]{
 -- Page 510, Offset 17838. Mods: @k, @e.
 -- 「ま、そうよね。志貴も疲れてるようだし、
-"I suppose so. You must be tired after all,
+"Well, I guess that makes sense. You must be tired.
 }
 [sha:055e9e1e500a8b104be9d16d013a6063a481cbe2]{
 -- Page 510, Offset 17839. Glued. Mods: @x.
 -- こわーい妹さんの目があるしね？」
- plus you have that scary little sister of yours to deal with, don't you?"
+ Plus you have that scary little sister of yours to deal with, don't you?"
```

Comment:
妹さん is consistently how Arcueid refers to Akiha (even to her face!) so maybe that should go in the notes.
It's not all too relevant in our case because it mostly comes up in KT and MB, but regardless.